frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```diff
@@ -403,47 +403,48 @@
       }
 
       .phone-controls {
-        border-radius: 40px;
-        border: $border-radius-xs solid $color-light-gray-20;
-        padding: $spacing-sm;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        gap: $spacing-md;
-
-        .forwarding-controls-button {
-          border-radius: $border-radius-lg;
-          background-color: $color-light-gray-30;
-          padding: $spacing-sm $spacing-sm;
+        .forwarding-toggle {
+          border-radius: 40px;
+          border: $border-radius-xs solid $color-light-gray-20;
+          padding: $spacing-sm;
           display: flex;
-          width: 100%;
           align-items: center;
           justify-content: center;
-
-          &:hover {
-            background: $color-purple-05;
-            color: $color-blue-50;
-
-            .forward-icon,
-            .block-icon {
-              color: $color-blue-50;
+          gap: $spacing-md;
+
+          .forwarding-toggle-icon {
+            border-radius: $border-radius-lg;
+            background-color: $color-light-gray-30;
+            padding: $spacing-sm $spacing-sm;
+            display: flex;
+            width: 100%;
+            align-items: center;
+            justify-content: center;
+
+            @media screen and #{$mq-md} {
+              width: $layout-xl;
             }
           }
 
-          @media screen and #{$mq-md} {
-            width: $layout-xl;
+          &.is-forwarding .forward-icon,
+          &.is-blocking .block-icon {
+            background-color: $color-violet-70;
+            color: $color-white;
           }
-        }
 
-        .active-button {
-          background-color: $color-violet-70;
-        }
+          &:hover {
+            border-color: $color-violet-70;
+          }
```

Comment:
I don't think it's necessary to have a hover state when there's already one for the individual button. I checked with Eduardo about this too.

https://user-images.githubusercontent.com/13066134/191796045-129e5725-1638-4a8f-915a-df51969a8741.mov

Reply:
Hmm, so I had it just for the individual button at first, but it got kinda confusing because the entire thing is the click target, not just the individual button - so you'd hover the currently-active "button", and then the other button would change background colour. What do you think? (I can also quickly push the change and you can see experience yourself if you want, see what you like better?)

Reply:
I think it's fine to just show a hover state for the inactive button since it's still some sort of interactive feedback for the user. Ideally this would be a literal toggle lol
<img width="117" alt="image" src="https://user-images.githubusercontent.com/13066134/191803495-0a7d07c5-bfcf-4402-8dc0-8e398e7e5c5e.png">

Reply:
OK, removed the border in 1c8de44c941bc712aa0ac23f69024edf57867da0

I did change the blue icon colour to only appear on hover and not on focus, because otherwise there was no difference between the hover and focus states (which would be confusing when you click it, then move your mouse away from the toggle - it would still be focused, so there would be no observable difference).

[Screencast from 23-09-22 16:57:37.webm](https://user-images.githubusercontent.com/4251/191990687-11050d29-5d32-4be0-99bb-41fec1137b23.webm)